satpy/etc/enhancements/generic.yaml

```diff
@@ -279,6 +279,21 @@ enhancements:
       kwargs:
         gamma: [2, 3, 2]
 
+  sar-ice-log:
```

Comment:
Can you put this in a sar-specific sensor YAML file?

Reply:
Probably want to add `sensor: sar` (or whatever the sensor name is) to this definition too.

Reply:
well, I could put it in a SAR-C file, but I'm pretty sure this composite would work also for other SAR sensors. Does it mean I would need to add an enhancement file for each sensor?

Reply:
Oh I don't understand how the sar sensor works. I guess I would assume it would be like VIIRS where the satellite is different but the sensor name is the same. If different models of the SAR instrument could use this and we identify them all differently then yeah you would need a different file for each sensor name...or leave it here in generic. I'm ok with either.

Reply:
I'm going to leave this as it is now, as we don't have a sensor-generic way of specifying the enhancements it seems. Like Sentinel 1's Sar-C and TerraSar-x's Sar-X instrument would both be candidates for this composite, but we can't put this in a single sar-family enhancement config-file.